testplan/report/testing/base.py

```diff
@@ -70,6 +70,9 @@ class Status(object):
     SKIPPED = 'skipped'
     READY = 'ready'
     RUNNING = 'running'
+    XFAIL = 'xfail'
+    XPASS = 'xpass'
+    XPASS_STRICT = 'xpass(strict)'
```

Comment:
What does XPASS_STRICT mean exactly? Personally I think that if a test is marked as strict xfail and it passes, we should just mark as FAIL.

Reply:
> What does XPASS_STRICT mean exactly? Personally I think that if a test is marked as strict xfail and it passes, we should just mark as FAIL.

Many users are using this feature to mark unstable testcase. The testcase not always fail or pass

Reply:
? I don't think that answers my question. I get what the `strict=True` flag to xpass does, I'm just not sure that we need a separate XPASS_STRICT status.

Reply:
> ? I don't think that answers my question. I get what the `strict=True` flag to xpass does, I'm just not sure that we need a separate XPASS_STRICT status.

OK, I misunderstand.  Let's mark is as FAIL